app/digital/ts/models/ioobjects/outputs/LED.ts

```diff
@@ -19,6 +25,57 @@ export class LED extends DigitalComponent {
         this.inputs.first.setTargetPos(V(0, 2*DEFAULT_SIZE));
     }
 
+    // @Override
+    public getMinPos(): Vector {
+        const min = V(Infinity);
```

Comment:
We need to use super.getMinPos() and super.getMaxPos for getMaxPos, so that the "find minimum pos for ports" doesn't have to be done in here.
See: https://github.com/OpenCircuits/OpenCircuits/pull/412/files